pkgs/by-name/ed/eduvpn-client/package.nix

```diff
@@ -38,7 +40,12 @@ python3Packages.buildPythonApplication rec {
     setuptools
   ];
 
-  patches = [ ./nix-python-prefix.patch ];
```

Comment:
I think it's best to keep this patch in

Reply:
I can revert this change, however, I was thinking that a more streamlined package with less nix specific patches would make future updates more convenient (and keep nixpkgs a bit slimmer)

Reply:
I agree with you, but this ensures the nix package would never clash with any other installs of eduvpn. E.g. through pip. For e.g. 4.4.0 I will think about how I can solve this better

Reply:
This may simplify stuff to much, but could you not put the prefix as the first element, such that the relative path to the running installation is prefered?

Reply:
I am thinking to do that but that breaks pip and local installs (git checkouts running `python3 -m eduvpn.ui`). So I would have to patch the pip package and find add a script to run it locally with a patch

Reply:
Just another note I will probably need to use `pkg_resources` module or so, I will try this for the next version.

Reply:
fair, do you think it is worth it to readd it for now or can we leave it as is, and simply hope that nothing bad happens until the new versions comes out  (also accounting for that probably only very few people install the same package using nix and pip)?

Reply:
as the intention is to remove it in the future, I would go for adding it. But my opinions on this are not super strong so if you feel like this is also good then we will go for this. Let me build the package and then approve

Reply:
great, then let's leave it like this. I am not an expert but I have a feeling that it is preferred to avoid unnecessary patches.